upgrades/2024_11_19_161132_create_permission_view_all_cases_all_requests.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+use ProcessMaker\Models\Permission;
+use ProcessMaker\Upgrades\UpgradeMigration as Upgrade;
+
+class CreatePermissionViewAllCasesAllRequests extends Upgrade
+{
+    private const NEW_NAME_GROUP = 'Cases and Requests';
+
+    private const PERMISSIONS = [
+        [
+            'name' => 'view-all_cases',
+            'title' => 'View All Cases',
+        ],
+        [
+            'name' => 'view-my_requests',
+            'title' => 'View My Requests',
+        ],
+    ];
+
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        // Update the group Requests to Cases and Requests
+        DB::table('permissions')
```

Comment:
Consider to verify the `view-all_requests` has the right label `View All Requests`

Reply:
@caleeli the observation was fixed